Tools/ros2/ardupilot_msgs/srv/ModeSwitch.srv

```diff
@@ -0,0 +1,15 @@
+
+# This service requests the vehicle to switch its drive/flight mode
+
+# mode : Set the value to the drive/flight mode to be used
+# Copter : https://github.com/ArduPilot/ardupilot/blob/master/ArduCopter/mode.h#L14 
+# Rover  : https://github.com/ArduPilot/ardupilot/blob/master/Rover/mode.h#L12
+# Plane  : https://github.com/ArduPilot/ardupilot/blob/master/ArduPlane/mode.h#L22
+
+int8 mode
+---
+# status    : True if the request for mode switch was successful, False otherwise
+# curr_mode : Returns the code for the current drive/flight mode , after the processing the request
+
+bool status
+int8 curr_mode
```

Comment:
Please add trailing newline so you `cat` the file without messing up your terminal.